x-pack/plugins/data_visualizer/public/application/index_data_visualizer/embeddables/field_stats/field_stats_factory.tsx

```diff
@@ -362,6 +366,47 @@ export const getFieldStatsChartEmbeddableFactory = (
             };
           }, []);
 
+          if (viewType === FieldStatsInitializerViewType.DATA_VIEW && !dataViews) {
+            return (
+              <EuiEmptyPrompt
+                color="primary"
+                title={
+                  <h3>
+                    <FormattedMessage
+                      id="xpack.dataVisualizer.dashboard.fieldStats.noDataViewSelected"
+                      defaultMessage="No data view selected"
+                    />
+                  </h3>
+                }
+                body={
+                  <p>
+                    <FormattedMessage
+                      id="xpack.dataVisualizer.dashboard.fieldStats.noDataViewSelectedDescription"
+                      defaultMessage="Pick a data view to view field statistics."
+                    />
+                  </p>
+                }
+              />
+            );
+          }
+
+          if (isEsqlMode && !isEsqlEnabled) {
```

Comment:
Nit - should this callout be the full width of the panel (add a Field Statistics panel, then disable ES|QL):

<img width="1269" alt="Screenshot 2024-09-20 at 16 56 50" src="https://github.com/user-attachments/assets/5fc5e2a7-4c7b-4f68-acce-66d08b77b545">

Reply:
Fixed here [`c6ded34` (#193587)](https://github.com/elastic/kibana/pull/193587/commits/c6ded34a4c979525fd3a7e9aef93bde8df60aa19)